start_integrate_test.sh

```diff
@@ -36,12 +36,12 @@ echo "github pull request base branch -> $3"
 echo "github pull request head branch -> ${GITHUB_HEAD_REF}"
 
 echo "use dubbo-go-samples $3 branch for integration testing"
-git clone -b main https://github.com/dubbo-go-pixiu/samples.git integrate_samples && cd integrate_samples
+git clone -b main https://github.com/apache/dubbo-go-pixiu-samples.git integrate_samples && cd integrate_samples
 
 # update dubbo-go to current commit id
 go mod edit -replace=github.com/apache/dubbo-go-pixiu=github.com/"$1"@"$2"
 
-grep -rl "github.com/apache/dubbo-go-pixiu/pkg" | xargs sed -i 's/github.com\/apache\/dubbo-go-pixiu\/pkg\//github.com\/apache\/dubbo-go-pixiu\/pixiu\/pkg\//g'
+#grep -rl "github.com/apache/dubbo-go-pixiu/pkg" | xargs sed -i 's/github.com\/apache\/dubbo-go-pixiu\/pkg\//github.com\/apache\/dubbo-go-pixiu\/pixiu\/pkg\//g'
```

Comment:
why delete it

Reply:
apache/dubbo-go-pixiu-samples.git has imported apache/dubbo-go-pixiu/pixiu/pkg